tests/backup/backup_test.go

```diff
@@ -6353,7 +6353,7 @@ var _ = Describe("{DeleteUsersRole}", func() {
 			log.InfoD("Creating %d users", numberOfUsers)
 			var wg sync.WaitGroup
 			for i := 1; i <= numberOfUsers; i++ {
-				userName := fmt.Sprintf("autouser%v", i)
+				userName := fmt.Sprintf("testautouser%v", i)
 				firstName := fmt.Sprintf("FirstName%v", i)
 				lastName := fmt.Sprintf("LastName%v", i)
 				email := fmt.Sprintf("testuser%v@cnbu.com", i)
```

Comment:
We are seeing user creation failure because of email uniqueness check. Please append the email as well with time stamp. Else we see these kind of errors in the response:

```
"errorMessage": "User exists with same email"
```